README.md

```diff
@@ -1,5 +1,7 @@
 # Mereswine
 
+![alt text](mereswine/static/logo.png)
```

Comment:
please don't leave "alt text" there :) either put a proper alt text or leave it empty

Reply:
I'd also put an absolute URL to the image on github